colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -318,7 +320,9 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or dataset versioning. To navigate to the relevant informationm, click this [link](https://wandb.ai/wandb/artifact-basics/overview), then click on the **Artifacts** tab."
+        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or dataset versioning. To navigate to the relevant information, click this [link](https://wandb.ai/wandb/artifact-basics/overview), then click on the **Artifacts** tab.\n",
```

Comment:
> This can give you insight into your model's performance or dataset versioning. 
can we be more specific here on what we're pointing the user to look at in web app? what does model performance mean here - is this the metrics logged to the run? is it to see different versions? This is vague and not too helpful to the user once they get to that link